### ISSUE TITLE
Pages are not listed or imported

### ISSUE THREAD

**Issue by ironprogrammer** [WordPress/tumblr-importer#2]
During testing for #1, the importer did not list or import custom Tumblr blog pages. TBH, I don't know when/if this worked properly, but since it's a column listed on the importer, I would expect it to do so.

## Steps to Reproduce
1. In Tumblr, click the Account icon, and under the blog section, select "Edit Appearance".
2. In the Custom Theme section, click the **Edit theme** button.
3. In the theme editor, scroll down and click "+ Add a page". Give the page a path, a name, and sample content, then click **Save**.
4. Click the arrow in the upper-left to return to the theme editor, and click **Exit** to leave the customizer.
5. In WordPress, navigate to *Tools > Import*, install the Tumblr importer if needed, and then run it. Follow the steps outlined in #1 to create a Tumblr app, and to link your WordPress site to Tumblr.
6. Observe that the "Pages Imported" column lists "0", and running the importer does not update this value, nor import pages into WordPress.

<img width="778" alt="Tumblr importer does not list pages" src="https://user-images.githubusercontent.com/824344/207428169-5a97e3ac-43fd-471a-9f8e-5d5e6ac06cb4.png">

### Supplemental Assets

| | |
|:-:|:-:|
| <img width="200" alt="Account menu" src="https://user-images.githubusercontent.com/824344/207428575-abbc5501-0213-47b6-a0e8-e3c366227513.png"> | <img width="500" alt="Custom Theme" src="https://user-images.githubusercontent.com/824344/207428665-ea3180a8-3791-4ba7-856d-9c0897e3e498.png"> <hr> <img width="260" alt="Theme editor" src="https://user-images.githubusercontent.com/824344/207429127-bd457c07-44f2-4a50-84cd-c6c0555697c4.png"> <hr> <img width="500" alt="Add a page" src="https://user-images.githubusercontent.com/824344/207428870-4462dd09-6cfb-493b-b4c9-b03d2de4515b.png"> |

**Comment by zaerl** [comment]
@ironprogrammer Yeah, I noticed. It didn't work since the introduction of the new Tumblr API. Unfortunately [there is no way to get the pages](https://github.com/WordPress/tumblr-importer/blob/trunk/tumblr-importer.php#L383). I think that it's a good idea to remove that part. What do you think?

**Comment by ironprogrammer** [comment]
Yes, I agree that removing it would be clearer for users.

The Tumblr blog site functionality where pages live has a distinct "legacy" feel to it 😅. But to be certain, I've reached out to Tumblr to see if this endpoint is coming back, or if the feature should be removed for good.

**Comment by zaerl** [comment]
Good idea. Let's wait, then.

Removing that part is simple.
